src/plays/navbar/navBar.css

```diff
@@ -0,0 +1 @@
+/* enter stlyes here */
```

Comment:
Can we remove this file as it's an empty file?
we can add it later if we want to add some styles.